src/olympia/users/models.py

```diff
@@ -261,20 +261,6 @@ def is_artist(self):
             addon__type=amo.ADDON_PERSONA).exists()
 
     @property
-    def source(self):
```

Comment:
It was only used during the migration when we would prompt for email then conditionally go to email/password login or FxA.